### PR TITLE
*: add metrics for checking memory locks

### PR DESCRIPTION
**Pull request by sticnarf**
### What problem does this PR solve?

There was no metrics about how long checking memory locks takes. Usually it is fast, but it is still useful for diagnosis.

### What is changed and how it works?

Track all memory lock checking durations in histograms.

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

Raft Process:

图片

Scheduler-Get:

图片

Coprocessor Details:

图片

### Release note 

- No release note.

### CODE REVIEW

File: src/coprocessor/metrics.rs [hunk 1]
```diff
@@ -224,6 +233,16 @@ lazy_static! {
         "The number of tasks waiting for the semaphore"
     )
     .unwrap();
+    pub static ref MEM_LOCK_CHECK_HISTOGRAM_VEC: HistogramVec =
+        register_histogram_vec!(
+            "tikv_coprocessor_mem_lock_check_duration_seconds",
+            "Duration of memory lock checking for coprocessor",
+            &["result"],
+            exponential_buckets(1e-6f64, 4f64, 10).unwrap() // 1us ~ 1s
```
Comment (youjiali1995) [inline src/coprocessor/metrics.rs:241]: It's 1us\~262ms, not 1us\~1s.

**Review by youjiali1995 — COMMENTED** [review]
Rest LGTM

**Review by youjiali1995 — APPROVED** [review]

**Review by 5kbpers — APPROVED** [review]

**Review by MyonKeminta — APPROVED** [review]

**Pull request merged**